### PR TITLE
feat(frontend): sign-out functionality bb-241

### PR DESCRIPTION
**Pull request by ihorLysak**
### Summary

Implemented sign-out

https://github.com/user-attachments/assets/377c99b3-793e-4c76-922b-25ef91f775a9

### CODE REVIEW

File: apps/frontend/src/libs/components/button/button.tsx [hunk 1]
```diff
@@ -41,6 +41,7 @@ const Button: React.FC<Properties> = ({
 		<span
 			className={getValidClassNames(
 				hasVisuallyHiddenLabel && "visually-hidden",
+				variant === "sidebarItem" && styles["last"],
```
Comment (fshabanov) [inline apps/frontend/src/libs/components/button/button.tsx:44]: I don't think this should be a part of button. Should be handled in sidebar

File: apps/frontend/src/modules/auth/slices/auth.slice.ts [hunk 2]
```diff
@@ -53,7 +54,12 @@ const { actions, name, reducer } = createSlice({
 	},
 	initialState,
 	name: "auth",
-	reducers: {},
+	reducers: {
+		logOut: (state) => {
```
Comment (fshabanov) [inline apps/frontend/src/modules/auth/slices/auth.slice.ts:58]: Let's create a separate action for that

**Review by fshabanov — CHANGES_REQUESTED** [review]

File: apps/frontend/src/libs/components/button/styles.module.css [hunk 3]
```diff
@@ -35,10 +35,24 @@
...
+	background: none;
+	border-top: 1px solid var(--white);
+	border-radius: 0%;
+}
+
 .position-right {
 	right: 20px;
 }
 
+.last {
```
Comment (fshabanov) [inline apps/frontend/src/libs/components/button/styles.module.css:52]: I believe you can remove it now

File: apps/frontend/src/libs/components/sidebar/sidebar.tsx [hunk 4]
```diff
@@ -37,19 +49,25 @@ const Sidebar: React.FC<Properties> = ({
...
+								iconName={href === pathname ? active : inactive}
+								key={label}
+								label={label}
+								pathname={pathname}
+								to={href}
+							/>
+						);
+					})}
+				</div>
+				<button className={styles["signOut-btn"]} onClick={handleSignOut}>
```
Comment (fshabanov) [inline apps/frontend/src/libs/components/sidebar/sidebar.tsx:67]: Use our Button component

File: apps/frontend/src/modules/auth/slices/auth.slice.ts (hunk 2 above)
Comment (fshabanov) [reply apps/frontend/src/modules/auth/slices/auth.slice.ts:62]: Remove this one

**Review by fshabanov — CHANGES_REQUESTED** [review]

File: apps/frontend/src/libs/components/button/styles.module.css [hunk 5]
```diff
@@ -35,6 +35,16 @@
 	transform: translateY(-50%);
 }
 
+.sidebarItem-button {
```
Comment (fshabanov) [inline apps/frontend/src/libs/components/button/styles.module.css:38]: Do you use this style?

**Review by fshabanov — CHANGES_REQUESTED** [review]

File: apps/frontend/src/libs/components/button/styles.module.css (hunk 5 above)
Comment (ihorLysak) [reply apps/frontend/src/libs/components/button/styles.module.css:38]: that is strange, I remember removing it

**Review by fshabanov — APPROVED** [review]

**Pull request merged**